controller/WFCORE-433_git-persistence.adoc

```diff
@@ -0,0 +1,98 @@
+= Adding support for Git backend for configuration files
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
+
+== Overview
+
+Replace the current persistence mechanism for configuration files and history by one using a real version control system.
```

Comment:
Do you really mean "replace" or will the new system supplement the existing one?

Reply:
Supplement :)